tests/commons/src/main/java/org/codice/ddf/admin/comp/graphql/GraphQlHelper.java

```diff
@@ -80,6 +90,20 @@ public void waitForGraphQLSchema() {
             });
   }
 
+  public <T> GraphQlResponse<T> mutationRequest(
+      String usingMutation, String jsonResultPath, Map<String, Object> arguments) {
+    return sendAndExtract(createMutationRequest(usingMutation, arguments), jsonResultPath);
+  }
+
+  public <T> GraphQlResponse<T> queryRequest(
+      String usingQuery, String jsonResultPath, Map<String, Object> arguments) {
+    return sendAndExtract(createQueryRequest(usingQuery, arguments), jsonResultPath);
+  }
+
+  private <T> GraphQlResponse<T> sendAndExtract(GraphQLRequest request, String jsonPath) {
```

Comment:
Would it be easier to update the method `getResult` to the `GraphQlResponse` to take in a `jsonPath` so we don't need this method?

Reply:
This is a good idea. If the responses are complex we can get multiple jsonPath results from the response. 👍